includes/class-signature.php

```diff
@@ -242,6 +242,11 @@ public static function verify_http_signature( $request ) {
 	 * @return string $publicKeyPem
 	 */
 	public static function get_remote_key( $key_id ) { // phpcs:ignore
+		$pre = apply_filters( 'pre_get_remote_key', false, $key_id );
```

Comment:
Tried adding this filter to short-circuit the function during Tests, maybe someone else can get this to work